packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/quanticSearchBoxInput.js

```diff
@@ -246,26 +249,28 @@ export default class QuanticSearchBoxInput extends LightningElement {
   }
 
   onTextAreaInput() {
-    this.handleValueChange();
+    this.sendInputValueChangeEvent(this.input.value, true);
```

Comment:
This has changed because  `handleValueChange` was doing: 
```
this.sendInputValueChangeEvent(this.input.value, false);
```
The second parameter is named `resetSelection`, when it's true the hover on the suggestion list items will be reset.

When the value of the input changes, we need to reset the suggestions and that's why I'm updating this second parameter to `true`, it was causing an issue before.

Reply:
the thing is that `handleValueChange ` is an event listener in the default search box, so we can't really add a parameter to it.

Reply:
Also did a small refactor by removing `handleKeyValues ` to avoid duplicating the same logic twice.